src/schema/schema.js

```diff
@@ -236,7 +246,7 @@ var sortMixin = {
         },
         op: {
           type: 'string',
-          enum: ['avg', 'sum', 'min', 'max', 'count'],
+          enum: VALID_OPS,
```

Comment:
In practice, these depends on the type of the field referred, but since we don’t have `type` in sort property, it does not matter.